tests/acceptance/helpers/sharingHelper.js

```diff
@@ -342,13 +343,39 @@ module.exports = {
         .postOCS(apiURL, user)
         .then(res => {
           res = httpHelper.checkStatus(res, 'The response status is not the expected value')
+          if (client.globals.ocis) return res.text()
           return res.json()
         })
         .then(res => {
+          if (client.globals.ocis) {
+            if (res !== '') {
+              throw new Error(`
+              This is a good error,
```

Comment:
@dpakach can you explain why this might happen?

Reply:
Accepting a share in ocis doesn't gives any response right now.
When that is fixed and response is not empty then someone will hopefully see this message and can fix it.
More in https://github.com/owncloud/product/issues/207